codalab/lib/bundle_cli.py

```diff
@@ -815,8 +815,15 @@ def collapse_bare_command(argv):
         """
         try:
             i = argv.index('---')
-            # Convert the command after '---' to a shell-escaped version of the string.
-            shell_escaped_command = [quote(x) for x in argv[i + 1 :]]
+            # This is to get the default list of punctuation chars from shlex. When punctuation_chars = True,
+            # the class property punctuation_chars is set as '();<>|&'. Otherwise, an empty string will be returned.
+            punctuation_chars = list(
+                shlex.shlex(COMMAND_PLACEHOLDER, punctuation_chars=True).punctuation_chars
```

Comment:
Can't you initialize a `shlex.shlex` object with an empty string instead of a placeholder?

Reply:
Yes. We can initialize `shlex.shlex` with an empty string. I was trying to keep it this placeholder variable consistent with other placeholder variables, e.g. https://github.com/codalab/codalab-worksheets/blob/962189207c363bc26c76b5d3a6d765d687d8c70e/codalab/common.py#L12
But we can certainly change it to an empty string.

Reply:
It's a stylistic choice, whatever you prefer is fine. I was just curious.

Reply:
I think empty string would be better (add comment saying any string is fine) since we don't use the placeholder anywhere else.